### PR TITLE
[stable/pomerium] helm lint fixups

### LINKED ISSUE #34: Helm lint --strict fails
Hello guys,

So, I was playing with Pomerium using the helm charts. I change the configuration to fit my deployment needs, but when I tried to deploy the chart it failed.

I have a `helm lint --strict` stage in my pipeline which failed the deployment.

```
➜  pomerium helm lint --strict pomerium                                                                  
==> Linting pomerium
[ERROR] templates/: render error in "pomerium/templates/tls-secrets.yaml": template: pomerium/templates/_helpers.tpl:5:31: executing "pomerium.name" at <.Values.nameOverride>: map has no entry for key "nameOverride"

Error: 1 chart(s) linted, 1 chart(s) failed

➜  pomerium helm lint --strict pomerium                                                                  
==> Linting pomerium
[ERROR] templates/: render error in "pomerium/templates/secret.yaml": template: pomerium/templates/secret.yaml:1:18: executing "pomerium/templates/secret.yaml" at <.Values.config.exist...>: map has no entry for key "existingSecret"

Error: 1 chart(s) linted, 1 chart(s) failed
```

Some of these configurations are not present in the default values.yaml file, nor are they present in the [Readme](https://github.com/pomerium/pomerium-helm#configuration)

### PR DESCRIPTION
**Pull request by travisgroth**
#### Is this a new chart
No

#### What this PR does / why we need it:

Fixes strict helm linting errors

#### Which issue this PR fixes
  - fixes https://github.com/pomerium/pomerium-helm/issues/34

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/mychartname]`)

### CODE REVIEW

File: stable/pomerium/values.yaml [hunk 1]
```diff
@@ -17,8 +21,8 @@ config:
   policy: {}
 
 authenticate:
-  # fullnameOverride: authenticate
-  # nameOverride: authenticate
+  fullnameOverride: authenticate
+  nameOverride: authenticate
```
Comment (victornoel) [inline stable/pomerium/values.yaml]: that's unfortunate, I think we don't want those set by default…

Comment (travisgroth) [reply stable/pomerium/values.yaml]: Set with empty strings instead now.

File: stable/pomerium/Chart.yaml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: pomerium
-version: 4.1.1
+version: 4.1.2
 appVersion: 0.5.1
```
Comment (victornoel) [inline stable/pomerium/Chart.yaml:4]: shouldn't we take this opportunity to up the version here too @travisgroth?

Comment (travisgroth) [reply stable/pomerium/Chart.yaml:4]: Since this is coming in as a third party patch, I'll do that as a follow up.

**Comment by travisgroth** [comment]
I think the failing test is a prow problem?

```
I1215 11:49:10.867] Error: Could not get apiVersions from Kubernetes: unable to retrieve the complete list of server APIs: metrics.k8s.io/v1beta1: the server is currently unable to handle the request
```

**Comment by desimone** [comment]
Possibly related to https://github.com/helm/helm/issues/6361 ?

**Comment by travisgroth** [comment]
>Possibly related to helm/helm#6361 ?

Quite possible...I'm seeing other folks with confused `/retest` commands, though not the same error.  Other PRs have been merged in the last few hours, though.  I'm pretty sure the diff here shouldn't cause the error showing up, though.

**Comment by desimone** [comment]
🤷‍♂

**Pull request merged**